include/xeus-cpp/xinterpreter.hpp

```diff
@@ -16,7 +16,7 @@
 #include <string>
 #include <vector>
 
-#include <clang/Interpreter/Interpreter.h>
+#include "clang/Interpreter/CppInterOp.h" // from CppInterOp package
```

Comment:
Deos CppInterop place a header in the `clang/Interpreter` include directory?

Reply:
Yeah I was thinking about this too !

Reply:
Nope, it is not but that's the design of the package - we should change that in the new versions.

Reply:
Ah, wait, are we having a common place for all headers? If everything is under include then it copies it there…

Reply:
I am still confused by why `CppInterOp.h`, which is a header from the CppInterOp package is under `PREFIX/include/clang`.

Reply:
Yes, not a good choice. We will fix this in the next release of CppInterOp. Do we want to block progress because of that?

Reply:
> We will fix this in the next release of CppInterOp.

Thanks!